routes/serve-routes.js

```diff
@@ -52,19 +63,21 @@ function extractPageFromClaims (claims, pageNumber) {
   return pageOfClaims;
 }
 
-function determineTotalPages (totalClaims) {
-  if (totalClaims === 0) {
+function determineTotalPages (claims) {
```

Comment:
Should all of this logic be in `routes` itself? (I don't actually know Express paradigms)

Reply:
Agreed.  I moved it to the controller layer.